Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM golang:1.20.3-alpine AS build
 
 # Install Alpine Dependencies
 RUN apk update && \
-    apk add build-base clang upx
+    apk add build-base clang upx git
```

Comment:
@wojciechos command from Makefile `go build -ldflags="-X main.Version=$(shell git describe --tags)" -o build/juno ./cmd/juno/` was failing because there were no git binary in build image

Reply:
Not failing entirely I meant flag `-ldflags="..."` wasn't doing anything helpful